src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/jpa/entity/RecommendationEntity.kt

```diff
@@ -129,14 +126,18 @@ data class RecommendationModel(
   var roshSummary: RoshSummary? = null,
   val countersignSpoTelephone: String? = null,
   val countersignSpoExposition: String? = null,
-  var countersignSpoName: String? = null,
-  var acoCounterSignEmail: String? = null,
-  var spoCounterSignEmail: String? = null,
-  var countersignSpoDateTime: LocalDateTime? = null,
   val countersignAcoExposition: String? = null,
   val countersignAcoTelephone: String? = null,
-  var countersignAcoName: String? = null,
-  var countersignAcoDateTime: LocalDateTime? = null
+
+  var userNamePartACompletedBy: String? = null, // deprecated
```

Comment:
We do as the entity has to follow the structure in the json blob in the DB.
If I deleted these, Spring Data would notice that there are old records in the DB still existing with the old fields and error out. 

Ways around that are 
1) a data migration
2) refactor RecommendationEntity to use a JSONObject type or Map<String,String> instead of the current strongly type entity.

What do you think Darren? Shall I go with one of those options or leave it as it is for now?